script/Arcueid/Day 6/06_00_ARC06_4.txt

```diff
@@ -32,7 +32,7 @@ I know it's a common refrain that the sun sets early in autumn, but it really do
 [sha:159daee16d84567a8970eb933ff97da50297febd]{
 -- Page 325, Offset 14723.
 -- 食堂には俺と秋葉と、その後ろに控えた琥珀さんと、俺の後ろに控えた翡翠の姿がある。
-Akiha and I sit in the dining room.
+Akiha and I sat in the dining room.
```

Comment:
Why the tense change?

Reply:
everyone was there with no sitting sound triggering, I assume they were already having dinner